core/spec/models/spree/variant/pricing_options_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'spec_helper'
+
+describe Spree::Variant::PricingOptions do
+  subject { described_class.new }
+
+  context '.default_price_attributes' do
+    it 'can be passed into a WHERE clause on Spree::Prices' do
+      expect do
+        Spree::Price.where(described_class.default_price_attributes).to_a
```

Comment:
Expect an empty array instead, and test whether it returns stuff if we setup stuff for it.